lib/utils/correlation.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const UUID = require('node-libuuid');
+
+/**
+ * Create a correlation id
+ * @return {String}
+ */
+exports.create = function create() {
+  const id = UUID.v4();
+
+  Log.log('INFO', `Created correlation id ${id}`);
```

Comment:
I think I'll dump that line instead. When the manager is created we log the creation at `DEBUG` level. I'm going to promote that to `INFO` which covers this situation.